.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+ +sudo: false
+
+ +node_js:
+   - '5'
+   - '4'
+
+ cache:
+   directories:
+   - node_modules
+
+ after_script:
```

Comment:
is _coverage_ a script in package.json? if not, you either add it there or remove it from here